src/patternfly/components/DataList/data-list.scss

```diff
@@ -368,6 +404,15 @@
   &.pf-m-flex-5 { flex-grow: 5; }
 }
 
+// Data list text
+.pf-c-data-list__text {
```

Comment:
Careful with applying this class to `span` s. The text mods won't affect inline items, see: https://patternfly-pr-3159.surge.sh/documentation/core/components/datalist#text-modifiers-data-list-text. You'll need to set a `display` property on `__text` or only apply to block level elements.

Reply:
It's `display: block;` in the table, but you might be able to make it `inline-block` without any adverse effects?